api/src/components/flag/service.ts

```diff
@@ -0,0 +1,121 @@
+import * as client from 'lib/client';
+
+export const get = async (id: string) => {
+    const flags = await client.prisma.publicationFlags.findFirst({
+        include: {
+            user: {
+                select: {
+                    id: true,
+                    orcid: true,
+                    firstName: true,
+                    lastName: true,
+                    email: true,
```

Comment:
Remove email address as it may not be publicly accessible. Make the UI interface optional for email